x-pack/plugins/security_solution/public/management/services/exceptions_list/exceptions_list_api_client.ts

```diff
@@ -0,0 +1,216 @@
+    }));
+
+    return exceptionToUpdateCleaned as UpdateExceptionListItemSchema;
+  }
+
+  /**
+   * Returns a list of items with pagination params.
+   * It accepts the allowed filtering, sorting and pagination options as param.
+   */
+  async find({
+    perPage = MANAGEMENT_DEFAULT_PAGE_SIZE,
+    page = MANAGEMENT_DEFAULT_PAGE + 1,
+    sortField,
+    sortOrder,
+    filter,
+  }: Partial<{
+    page: number;
+    perPage: number;
+    sortField: string;
+    sortOrder: string;
+    filter: string;
+  }> = {}): Promise<FoundExceptionListItemSchema> {
+    await this.ensureListExists;
+    return this.http.get<FoundExceptionListItemSchema>(`${EXCEPTION_LIST_ITEM_URL}/_find`, {
+      query: {
+        page,
+        per_page: perPage,
+        sort_field: sortField,
+        sort_order: sortOrder,
+        list_id: [this.listId],
+        namespace_type: ['agnostic'],
+        filter,
+      },
+    });
+  }
+
+  /**
+   * Returns an item filtered by id
+   * It requires an id in order to get the desired item
+   */
+  async get(id: string): Promise<ExceptionListItemSchema> {
+    await this.ensureListExists;
+    return this.http.get<ExceptionListItemSchema>(EXCEPTION_LIST_ITEM_URL, {
+      query: {
+        id,
+        namespace_type: 'agnostic',
+      },
+    });
+  }
+
+  /**
+   * It creates an item and returns the created one.
+   * It requires a CreateExceptionListItemSchema object.
+   */
+  async create(exception: CreateExceptionListItemSchema): Promise<ExceptionListItemSchema> {
+    await this.ensureListExists;
+    this.checkIfIsUsingTheRightInstance(exception.list_id);
+    return this.http.post<ExceptionListItemSchema>(EXCEPTION_LIST_ITEM_URL, {
+      body: JSON.stringify(exception),
+    });
+  }
+
+  /**
+   * It updates an existing item and returns the updated one.
+   * It requires a UpdateExceptionListItemSchema object.
+   */
+  async update(exception: UpdateExceptionListItemSchema): Promise<ExceptionListItemSchema> {
+    await this.ensureListExists;
+    return this.http.put<ExceptionListItemSchema>(EXCEPTION_LIST_ITEM_URL, {
+      body: JSON.stringify(ExceptionsListApiClient.cleanExceptionsBeforeUpdate(exception)),
+    });
+  }
+
+  /**
+   * It deletes an existing item.
+   * It requires a valid item id.
+   */
+  async delete(id: string): Promise<ExceptionListItemSchema> {
+    await this.ensureListExists;
+    return this.http.delete<ExceptionListItemSchema>(EXCEPTION_LIST_ITEM_URL, {
+      query: {
+        id,
+        namespace_type: 'agnostic',
+      },
+    });
+  }
+
+  /**
+   * It returns a summary of the current list_id
+   * It accepts a filter param to filter the summary results using KQL filtering.
+   */
+  async summary(filter?: string): Promise<ExceptionListSummarySchema> {
+    await this.ensureListExists;
+    return this.http.get<ExceptionListSummarySchema>(`${EXCEPTION_LIST_URL}/summary`, {
+      query: {
+        filter,
+        namespace_type: 'agnostic',
+      },
+    });
+  }
+}
```

Comment:
FYI: one method I thought of that we can add is `sync hasData(): Promise<boolean>` - and it just checks that the list has data (any data) in it.  We use this allot in our pages to determine if the empty state should be shown. You don't need to include it here, I can add it in my PR.